weekly/_posts/2015-08-14-weekly.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: io.js와 Node.js 주간 뉴스 2015년 8월 14일
+author: io.js
+ref: io.js Week of August 14th
+refurl: -

```

Comment:
https://medium.com/node-js-javascript/io-js-and-node-js-news-august-14th-c712ff526488